### ISSUE TITLE
How to include a relationship while sending a request?

### ISSUE THREAD

**Issue by gadelkareem** [wopian/kitsu#418]
It looks like the only place the relations are used in the GET request. Any way to send a POST|PATCH request while including a relationship?

**Comment by wopian** [comment]
The first example here has `targetUser` and `user` relationships in PATCH. Same applies with POST.

https://github.com/wopian/kitsu/tree/master/packages/kitsu#examples-6

**Comment by gadelkareem** [comment]
You are right, I was missing the id thanks!

**Comment by gadelkareem** [comment]
It seems that including an array of objects as a relationship, results in removing all the attributes of each object except the ID, ex:
```js
api.create('posts', {
      content: 'Hello World',
      users: [{
        id: '1',
        test: 'test1',
        type: 'users'
      },
      {
        id: '2',
        test: 'test2',
        type: 'users'
      }]
    })
```

Result
```json
{
  "data": {
    "type": "posts",
    "attributes": {
      "content": "Hello World"
    },
    "relationships": {
      "targetUser": {
        "data": {
          "id": "42603",
          "type": "users"
        }
      },
      "users": {
        "data": [
          {
            "id": "1",
            "type": "users"
          },
          {
            "id": "2",
            "type": "users"
          }
        ]
      }
    }
  }
}
```

**Comment by wopian** [comment]
Fixed `9.0.x` regression in `9.1.5`.
